assets/models/system/Mixtral-8x7B-v0-1/spec.yaml

```diff
@@ -20,8 +23,17 @@ tags:
       Standard_ND96asr_v4,
     ]
   inference_supported_envs:
-  - vllm
+    - vllm
```

Comment:
Did we test inference for finetuned model?

Reply:
I will test it on staging model, the older model MLmodel does not have base image.